src/components/fx/layout_attributes.js

```diff
@@ -101,5 +101,17 @@ module.exports = {
             ].join(' ')
         },
         editType: 'none'
+    },
+    selectdirection: {
+        valType: 'enumerated',
+        role: 'info',
+        values: ['h', 'v', 'd', 'any'],
+        dflt: 'any',
+        description: [
+            'When "dragmode" is set to "select", this limits the selection of the drag to',
+            'horizontal, vertical or diagonal. "h" only allows horizontal selection,',
+            '"v" only vertical, "d" only diagonal and "any" sets no limit.'
```

Comment:
Nice, I like `'d'`, I guess what it allows that `'any'` doesn't is selecting regions that are very narrow in height or width, which would otherwise devolve to pure horizontal or vertical selections. Cool.